CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### Under the hood
 - Add `build` RPC method, and a subset of flags for `build` task ([#3595](https://github.com/dbt-labs/dbt/issues/3595), [#3674](https://github.com/dbt-labs/dbt/pull/3674))
+- Switch to full reparse on partial parsing exceptions. Log and report exception information. ([#3725](https://github.com/dbt-labs/dbt/issues/3725), [#3733](https://github.com/dbt-labs/dbt/pull/3733))
```

Comment:
Let's make a new section for 0.20.2, above 0.20.1 and below 0.21.0b1, and put this there

Reply:
Done.